lib/backends/webgl/glsl-coordinate-lib.ts

```diff
@@ -1190,6 +1190,14 @@ export class CoordsGlslLib extends GlslLib {
       result[funcName] = new GlslLibRoutine(
           this.getValueFromSingle(name, rank, layout.width, layout.height, true),
           [`shapeUtils.indicesToOffset${funcName}`, `coordinates.offsetToCoords`, `fragcolor.getColorAsFloat`]);
+      funcName = `_${name}_Pack`;
```

Comment:
Just a reminder, we removed this change in conv_pack PR.

Reply:
Thanks. Reverted.